components/OCFBanner.tsx

```diff
@@ -200,7 +199,7 @@ export function OCFBannerWithData(props: OCFBannerProps) {
   }
 
   if (query.error) {
-    return <MessageBoxGraphqlError error={query.error} />;
```

Comment:
What is the context for this change?

Reply:
The query would throw due to the fact the `accountSlug` is invalid. I'm not sure if this is still required, I'll test it again and see what happens.

Reply:
It is not needed anymore, it was just a development workaround. Dropped the commit.